src/javascript/ast-value.ts

```diff
@@ -196,4 +197,23 @@ export function getIdentifierName(node: estree.Node): string|undefined {
   }
 }
 
+/**
+ * Formats the given identifier name under a namespace, if one is mentioned in
+ * the commentedNode's comment. Otherwise, name is returned.
+ */
+export function getNamespacedIdentifier(
+    name: string, docs?: jsdoc.Annotation): string {
+  if (!docs) {
+    return name;
+  }
+  const namespace = jsdoc.getTag(docs, 'memberof', 'description');
+  if (namespace) {
+    const rightMostIdentifierName = name.substring(name.lastIndexOf('.') + 1);
```

Comment:
Yea there's definitely some places where that would be nice, but it may be too restrictive for formatting things like `namespaces` that aren't actually node's.